Source/santad/SNTExecutionController.mm

```diff
@@ -268,12 +268,11 @@ - (void)validateExecEvent:(const Message &)esMsg postAction:(bool (^)(SNTAction)
   // TODO(markowsky): Maybe add a metric here for how many large executables we're seeing.
   // if (binInfo.fileSize > SomeUpperLimit) ...
 
-  SNTCachedDecision *cd = [self.policyProcessor decisionForFileInfo:binInfo
-      targetProcess:targetProc
-      configState:configState
-      preCodesignCheckCallback:^(void) {
-        esMsg.UpdateStatState(StatChangeStep::kCodesignValidation);
-      }
+  SNTCachedDecision *cd = [self.policyProcessor
+             decisionForFileInfo:binInfo
+                   targetProcess:targetProc
+                     configState:configState
+        preCodesignCheckCallback:nil
```

Comment:
WDYT about removing this callback?

Reply:
Oh whoops, that didn't make the commit, needs to be removed cause won't build otherwise. Fixed.